src/custom/pages/NewSwap/helpers/tradeReceiveAmount.ts

```diff
@@ -0,0 +1,32 @@
+import { TradeType } from '@uniswap/sdk-core'
+import { formatSmartAmount } from 'utils/format'
+import TradeGp from 'state/swap/TradeGp'
+
+export interface ReceiveAmountInfo {
+  type: 'from' | 'to'
+  amountBeforeFees: string
+  amountAfterFees: string
+  feeAmount: string
+}
+
+export function getInputReceiveAmountInfo(trade: TradeGp): ReceiveAmountInfo {
+  return {
+    type: 'from',
+    // TODO: why we don't check the same for OUTPUT currency?
```

Comment:
Because fees are always taken from INPUT currency

Reply:
Thank you very much!